utils/clflags.ml

```diff
@@ -552,6 +552,176 @@ let set_save_ir_after pass enabled =
   in
   save_ir_after := new_passes
 
+module Dump_option = struct
+  type t =
+    | Source
+    | Parsetree
+    | Typedtree
+    | Shape
+    | Raw_lambda
+    | Lambda
+    | Instr
+    | Raw_clambda
+    | Clambda
+    | Raw_flambda
+    | Flambda
+    | Cmm
+    | Selection
+    | CSE
+    | Live
+    | Spill
+    | Split
+    | Interf
+    | Prefer
+    | Regalloc
+    | Scheduling
+    | Linear
+    | Interval
+
+  let compare (op1 : t) op2 =
+    Stdlib.compare op1 op2
+
+  let to_string = function
+    | Source -> "source"
+    | Parsetree -> "parsetree"
+    | Typedtree -> "typedtree"
+    | Shape -> "shape"
+    | Raw_lambda -> "raw_lambda"
+    | Lambda -> "lambda"
+    | Instr -> "instr"
+    | Raw_clambda -> "raw_clambda"
+    | Clambda -> "clambda"
+    | Raw_flambda -> "raw_flambda"
+    | Flambda -> "flambda"
+    | Cmm -> "cmm"
+    | Selection -> "selection"
+    | CSE -> "cse"
+    | Live -> "live"
+    | Spill -> "spill"
+    | Split -> "split"
+    | Interf -> "interf"
+    | Prefer -> "prefer"
+    | Regalloc -> "regalloc"
+    | Scheduling -> "scheduling"
+    | Linear -> "linear"
+    | Interval -> "interval"
+
+  let of_string = function
+    | "source" -> Some Source
+    | "parsetree" -> Some Parsetree
+    | "typedtree" -> Some Typedtree
+    | "shape" -> Some Shape
+    | "raw_lambda" -> Some Raw_lambda
```

Comment:
Could you make it accept `"rawlambda"` too (and the same for raw_clambda and raw_flambda) ? It would be more natural for people knowing the `-drawlambda` flag, I think.

Reply:
Right, I'll just replace `raw_foo` by `rawfoo`.